docs/ibc/light-clients/overview.md

```diff
@@ -1,3 +1,45 @@
 <!--
 order: 1
--->
\ No newline at end of file
+-->
+
+# Overview 
+
+Learn how to build IBC lightclient modules and fulfill the interfaces required to integrate with core IBC. {synopsis}
+
+The following aims to provide a high level IBC lightclient module developer guide. The lifecycle of IBC lightclients are created and maintained by the core IBC `MsgServer` and the abstractions provided by the `02-client` submodule. A lightclient module developer is only required to implement the methods and fulfil the expectations of a defined set of interfaces. In ibc-go these interfaces reside within the `core/modules/exported` package. 
```

Comment:
Maybe something like this instead?
```suggestion
The following aims to provide a high level IBC light client module developer guide. Access to IBC light clients are gated by the core IBC `MsgServer` which utilizes the abstractions set by the `02-client` submodule to call into a light client module. A light client module developer is only required to implement the `ClientState` and `ConsensusState` interfaces as defined in the `core/modules/exported` package. 
```
Should probably also link to the interfaces

Reply:
Thanks, I will use this and adapt it slightly. I guess this part is actually incorrect:
> A light client module developer is only required to implement the `ClientState` and `ConsensusState` interfaces as defined in the `core/modules/exported` package. 

`ClientMessage` also needs to be implemented for updates and misbehaviour